crates/rattler_solve/src/libsolv/mod.rs

```diff
@@ -1,25 +1,37 @@
+use rattler_conda_types::RepoDataRecord;
 use std::collections::HashMap;
 
 mod wrapper;
 
-use crate::{PackageOperation, SolveError, SolverBackend, SolverProblem};
-use wrapper::flags::{SolvableFlags, SolverFlag};
-use wrapper::pool::{Intern, Pool, Verbosity};
-use wrapper::queue::Queue;
+use crate::{SolveError, SolverBackend, SolverProblem};
+use wrapper::{
+    flags::SolverFlag,
+    pool::Intern,
+    pool::{Pool, Verbosity},
+    solve_goal::SolveGoal,
+};
 
 /// A [`SolverBackend`] implemented using the `libsolv` library
 pub struct LibsolvSolver;
 
 impl SolverBackend for LibsolvSolver {
-    fn solve(&mut self, problem: SolverProblem) -> Result<Vec<PackageOperation>, SolveError> {
+    fn solve(&mut self, problem: SolverProblem) -> Result<Vec<RepoDataRecord>, SolveError> {
         // Construct a default libsolv pool
         let pool = Pool::default();
 
         // Setup proper logging for the pool
         pool.set_debug_callback(|msg, flags| {
             tracing::event!(tracing::Level::DEBUG, flags, "{}", msg);
         });
-        pool.set_debug_level(Verbosity::Low);
+        pool.set_debug_level(Verbosity::Extreme);
```

Comment:
Is this something we want to keep or were you using it for debugging and forgot to remove it? I have no preference, just curious

Reply:
Nice catch!